### ISSUE TITLE
[BUG] Build errors when implementing InAppMessagingDisplayDelegate

### ISSUE THREAD

**Issue by professorbk** [firebase/firebase-ios-sdk#4261]
### [REQUIRED] Describe your environment

  * Xcode version: 10.1
  * Firebase SDK version: 5.8.1
  * Firebase Component:         
        pod 'Firebase/Core'
        pod 'Firebase'
        pod 'Firebase/InAppMessagingDisplay'
        pod 'Firebase/DynamicLinks'
        pod 'Firebase/Messaging', '5.8.1'
        pod 'Fabric', ~> 1.10.2
        pod 'Crashlytics', ~> 3.13.2
        pod 'GoogleAppMeasurement', ~> 5.1.0

### [REQUIRED] Describe the problem
I installed FIAM, and can received in-app message. Then I wanted to implement InAppMessagingDisplayDelegate follows this guide:
https://firebase.google.com/docs/in-app-messaging/modify-message-behavior?platform=ios
There are errors when I build project:
**-Value of type 'InAppMessaging' has no member 'delegate'; did you mean 'release'? 
   -Use of undeclared type 'InAppMessagingDisplayDelegate'**
I did import FirebaseInAppMessaging
I tried to clean build, pod update, reopen Xcode but those errors still happen

**Comment by rizafran** [comment]
Hi @professorbk Looks like you're still using an older version of Firebase SDK. Please update your SDK to the [latest one](https://firebase.google.com/support/release-notes/ios). If the issue persists, you may provide a [minimal repro](https://stackoverflow.com/help/mcve) that I can run locally to replicate the issue.

**Comment by professorbk** [comment]
@rizafran thank you very much for your support. problem was resolved after I updated latest Firebase SDK.
InAppMessagingDisplayDelegate is available since FirebaseInAppMessaging 0.13.0 (mine was 0.11.0). 

I think we should add a note for the below guide, something like "InAppMessagingDisplayDelegate requires Firebase version x.x.x or higher". 
https://firebase.google.com/docs/in-app-messaging/modify-message-behavior?platform=ios
